server/models/groups/removeModerator.js

```diff
@@ -52,7 +52,21 @@ function removeModerator({ params, body, username }) {
                     'Either the person is already not a moderator or no such person found in the group!'
                   )
                 }
-                return resolve('Successfully removed from moderator!!')
+                const port = process.env.FRONTEND_PORT || 3000
+                const host = process.env.FRONTEND_HOST || 'localhost'
+                const body = {
+                  heading: `Removed from moderator of ${groupName}`,
+                  description: `You are no longer a moderator of the group [${groupName}](http://${host}:${port}/groups/${groupId})`,
```

Comment:
In this, I think the moderator will be able to remove himself (as moderator) ....would that be right?
and one more thing...do we need to show "removed moderator by whom"? (like removed admin by xyz in whatsapp)

Reply:
Whatsapp generally shows "You are no longer an admin"

Reply:
For now, we can allow moderator to remove himself (as moderator) . Later we can change it, if required.